packages/ra-core/src/i18n/Translate.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { TestTranslationProvider } from './TestTranslationProvider';
+import { Translate } from './Translate';
+
+export default {
+    title: 'ra-core/i18n/Translate',
+};
+
+export const Basic = () => (
+    <TestTranslationProvider
+        messages={{
+            custom: {
+                myKey: 'My Translated Key',
+            },
+        }}
+    >
+        <Translate i18nKey="custom.myKey" />
+    </TestTranslationProvider>
+);
+
+export const NoTranslation = () => (
+    <TestTranslationProvider messages={{}}>
+        <Translate i18nKey="custom.myKey" />
+    </TestTranslationProvider>
+);
+
+export const NoTranslationWithChildren = ({
+    isChildrenANode = false,
+}: {
+    isChildrenANode?: boolean;
+}) => (
+    <TestTranslationProvider messages={{}}>
+        <Translate i18nKey="custom.myKey">
+            {isChildrenANode ? (
+                <div style={{ color: 'red' }}>
+                    <i>My Default Translation</i>
+                </div>
+            ) : (
+                'My Default Translation'
+            )}
+        </Translate>
+    </TestTranslationProvider>
+);
+
+NoTranslationWithChildren.args = { isChildrenANode: false };
+NoTranslationWithChildren.argTypes = {
+    isChildrenANode: { control: 'boolean' },
+};
+
+export const Args = () => (
+    <TestTranslationProvider
+        messages={{
+            custom: {
+                myKey: ({ price }) => `It cost ${price}.00 $`,
```

Comment:
It's weird to use the function syntax - we never use it elsewhere. Why don't you use the interpolation syntax instead?

Reply:
My bad, I saw in the previous PR that it's not possible.